app/src/main/java/com/spascoding/taskycourse/feature_agenda/presentation/utils/Month.kt

```diff
@@ -0,0 +1,40 @@
+package com.spascoding.taskycourse.feature_agenda.presentation.utils
+
+import java.util.Calendar
+
+enum class Month(val description: String) {
+    JANUARY("JANUARY"),
+    FEBRUARY("FEBRUARY"),
+    MARCH("MARCH"),
+    APRIL("APRIL"),
+    MAY("MAY"),
+    JUNE("JUNE"),
+    JULY("JULY"),
+    AUGUST("AUGUST"),
+    SEPTEMBER("SEPTEMBER"),
+    OCTOBER("OCTOBER"),
+    NOVEMBER("NOVEMBER"),
+    DECEMBER("DECEMBER");
```

Comment:
This does not allow localization, I would recommend using the official LocalDate API:
```
LocalDate.now().month.getDisplayName(TextStyle.SHORT, Locale.US)
```